recipes/newrelic/infrastructure/ohi/rabbitmq/debian.yml

```diff
@@ -37,24 +41,153 @@ preInstall:
       - RabbitMQ Management Plugin is configured
       - RabbitMQ command line tool, rabbitmqctl, is in the PATH of the root user
   requireAtDiscovery: |
-    PLUGIN_EXIST=$(sudo cat /etc/rabbitmq/enabled_plugins | grep "rabbitmq_management" | wc -l)
-    if [ $PLUGIN_EXIST -eq 0 ]; then
+    MGMT_PLUGIN_PRESENT=$(sudo cat /etc/rabbitmq/enabled_plugins | grep "rabbitmq_management" | wc -l)
+    RABBITMQCTL_IN_USER_PATH=$([[ $(which rabbitmqctl 2>/dev/null | wc -l) -eq 1 ]] && echo "YES" || echo "NO")
+    RABBITMQCTL_IN_DEFAULT_PATH=$([[ $(find /usr/sbin -name rabbitmqctl 2>/dev/null | wc -l) -eq 1 ]] && echo "YES" || echo "NO")
+    if [[ "$MGMT_PLUGIN_PRESENT" -eq 0 || ( $RABBITMQCTL_IN_USER_PATH == "NO" && $RABBITMQCTL_IN_DEFAULT_PATH == "NO" ) ]]; then
       exit 132
     fi
```

Comment:
@amolero-nr maybe add an explicit exit 0 (with a comment?) to make it clear this is the case the installation supports?

Reply:
Thanks! Just added it.